templates/patterns/card/card.ui_patterns.yml

```diff
@@ -10,47 +10,81 @@ card:
       description: 'Card with special behaviour for print search results items.'
   fields:
     orientation:
+      # Full type: `'horizontal'|'vertical'`.
       type: string
       label:  'Orientation'
       description: 'Can be set to horizontal or vertical. Defaults to vertical.'
     title:
+      # Full type: `string|render|markup`.
+      # Markup should be avoided, except for links.
       type: text
       label: 'Title'
-      description: 'Card title.'
+      description: 'Card title. Supports markup, e.g. for links.'
       preview: 'Title card'
+    url:
+      type: string
+      label: 'Url (optional)'
+      description: |
+        If present, the title is displayed as a link.
+        Only use this if the title is not already a link.
+      preview: 'https://www.example.com/'
     subtitle:
+      # Full type: `string|render|markup`.
+      # Markup should be avoided if possible.
       type: text
       label: 'Subtitle'
       description: 'Card subtitle.'
       preview: 'Subtitle card'
     text:
+      # Full type: `string|render|markup`.
+      # Markup can be used e.g. for <p> tags.
       type: text
       label: 'Text'
       description: 'Card text.'
       preview: 'Text card with longer text which is supported'
     image:
+      # Full type: `array{path: string, alt: string, ..}`.
+      # Look for '_image' in 'bcl-card.html.twig'.
       type: array
       label: 'Image'
       description: 'Card image with path, alt and position.'
       preview:
         path: 'https://placeimg.com/1000/500/people'
         alt: 'Alternative text for card image'
+    meta:
+      # Full type: `list<text|render|markup>`.
+      # Markup should be avoided if possible.
+      type: array
+      label: 'Meta items'
+      description: 'Date or other meta information.'
+      preview:
+        - 'Organisation'
+        - 'Position'
     content:
+      # Full type: `string|render|markup`.
+      # This is meant to be the most flexible parameter.
+      # Markup can be used for buttons or other elements.
       type: render
       label: 'Content'
       description: 'Card content.'
       preview: '<button class="btn btn-primary">Custom action</button>'
     header:
+      # Full type: `string|render|markup`.
+      # Markup should be avoided if possible.
       type: text
       label: 'Card header'
       description: 'Card header content.'
       preview: 'Header of card'
     footer:
+      # Full type: `string|render|markup`.
+      # Markup should be avoided if possible.
       type: text
       label: 'Card footer'
       description: 'Card footer content.'
       preview: 'Footer of card'
     badges:
+      # Full type: `list<array{label: string|render|markup, ..}>`.
+      # See 'bcl-badge.html.twig'.
       type: array
       label: 'Badges'
       description: 'List of badges for the card.'
```

Comment:
The badges label are primary and secondary but has the same aspect, so I will change the label or add the badge style to the badges.

Reply:
I am confused by the wording of this comment.

Reply:
I think this is good now.